src/actions/plugins.js

```diff
@@ -6,11 +6,14 @@ import {bindClientFunc} from './helpers';
 
 import {PluginTypes} from 'action_types';
 
-export function getMarketplacePlugins() {
+export function getMarketplacePlugins(filter = '') {
```

Comment:
better to leave this as is and fix the `null` issue where we build the query params

Reply:
Done